scientific_library/tvb/simulator/simulator.py

```diff
@@ -38,7 +38,7 @@
 .. moduleauthor:: Paula Sanz Leon <Paula@tvb.invalid>
 
 """
-
+import sys
```

Comment:
shouldn't need `sys` in this module

Reply:
Yes. It is needed only for the progression messages.